motoko/hello-angular-motoko/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "hello-angular-motoko",
+  "version": "0.0.3",
+  "author": {
+    "name": "Shawn Dotey",
+    "email": "insight431@protonmail.com",
+    "url": "https://forum.dfinity.org/u/infinityio.tech"
+  },
+  "description": "An example of a Motoko project with an Angular 14 frontend",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "TypeScript",
+    "Angular",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "",
+    "prebuild": "",
+    "prestart": "",
+    "init:dfx": "npm run restart:clean:dfx && npm run refresh:motoko && dfx stop",
+    "start": "npm run restart:dfx && npm run refresh:motoko && npm run serve:both && npm run print:ng",
+    "test": "npm run restart:dfx && ng test",
+    "deploy:dfx": "npm run restart:dfx && npm run build:prod && dfx deploy --network ic --no-wallet",
+    "serve:both": "concurrently -k \"npm run watch:motoko\" \"ng serve && dfx stop\"",
+    "watch:motoko": "nodemon --watch src --ext mo -x \"npm run refresh:motoko\"",
+    "print:dfx:www": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id www)",
+    "print:dfx:motoko": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id motoko)",
+    "generate:motoko": "dfx generate motoko",
+    "restart:dfx": "dfx stop && dfx start --background",
+    "restart:clean:dfx": "dfx stop && dfx start --background --clean",
+    "refresh:motoko": "dfx deploy motoko && npm run copy:dfx",
+    "copy:dfx": "rsync -avr .dfx/$(echo ${DFX_NETWORK:-'**'})/canisters/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --exclude='*.most' --exclude='*.ts' --delete src/declarations",
+    "build:prod": "export NODE_ENV=production && ng build --configuration production",
+    "build:dev": "export NODE_ENV=development && ng build --configuration development"
```

Comment:
Does this mean the normal `dfx build --network <ic/local>` will not work for this project?

Reply:
`dfx new hello && cd hello && dfx deploy` builds and deploys a front end as well, no?

Reply:
Yes, it's uses the `webpack` command in the `build` npm script. I should be able to rework this so that we can use the `dfx deploy` instead of `npm run dfx:deploy` seeing that the `dfx deploy` method is preferred.